src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -32,7 +37,9 @@ final case class EServicesApiServiceImpl(
   attributeRegistryManagementService: AttributeRegistryManagementService,
   catalogProcessService: CatalogProcessService,
   tenantManagementService: TenantManagementService,
-  partyProcessService: PartyProcessService
+  partyProcessService: PartyProcessService,
+  catalogManagementService: CatalogManagementService,
```

Comment:
May I suggest using the catalog process to get document metadata?
This means changing catalog process response from File to document metadata.